var/spack/repos/builtin/packages/openstf/package.py

```diff
@@ -14,7 +14,9 @@ class Openstf(MakefilePackage):
     homepage = "http://www.e-em.co.jp/OpenSTF/"
     url      = "http://www.e-em.co.jp/OpenSTF/OpenSTF.zip"
 
-    version('1.3.1', sha256='4c39c81f70e3f8017fcb9cd457436c77d29e016d78bc5337f152f2eb078aa7b6')
+    version('1.7.0', sha256='50a0406dd452e3ec09d29aa362a426ca04c8a586d817ab1295988f578baeac2a')
+    version('1.3.1', sha256='4c39c81f70e3f8017fcb9cd457436c77d29e016d78bc5337f152f2eb078aa7b6',
+            url='http://www.e-em.co.jp/OpenSTF/old/OpenSTF_131.zip')
```

Comment:
Is the new version available at a version-specific URL too? If it is, we should define a `url_for_version`.

Reply:
can not.
The new version download file name does not include version information.